api/src/main/thrift/org/apache/aurora/gen/api.thrift

```diff
@@ -197,6 +197,15 @@ struct DockerParameter {
   2: string value
 }
 
+struct Variable {
+	1: string name
+	2: string value
+}
+
+struct Instance {
```

Comment:
I don't think the name is proper here. This is not a real representation of an instance, but just the collection of the differences between them.